src/strategies/qipowah/index.ts

```diff
@@ -0,0 +1,64 @@
+import { getProvider, getSnapshots } from '../../utils';
+import strategies from '..';
+
+export const author = 'royalaid';
+export const version = '1.0.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const promises: any = [];
+  const blocks = await getSnapshots(
+    network,
+    snapshot,
+    provider,
+    options.ERC20s.map((s) => s.network || network)
+  );
+
+  for (const ERC20 of options.ERC20s) {
+    if (
+      options.startBlocks &&
+      blocks[ERC20.network] < options.startBlocks[ERC20.network]
+    ) {
+      continue;
+    }
+
+    promises.push(
+      strategies['pagination'].strategy(
+        space,
+        ERC20.network,
+        getProvider(ERC20.network),
+        addresses,
+        {
+          limit: 120,
+          strategy: {
+            symbol: 'QiPowah',
+            name: 'erc20-balance-of',
+            params: {
+              symbol: 'QiPowah',
+              address: ERC20.address,
+              decimals: ERC20.decimals
+            }
+          }
+        },
+        blocks[ERC20.network]
+      )
+    );
+  }
+
+  const results = await Promise.all(promises);
+  return results.reduce((finalResults: any, strategyResult: any) => {
```

Comment:
Hey @royalaid  i think you are just adding 8 erc20 tokens balances right, you can set 8 strategies in your space and output will be sum of them

Reply:
We've been having issues with nested queries because some of our balance contracts timeout. This strategy helps declutter the configuration so we can quickly update it

Reply:
There is actually a limit on network calls sent from a strategy, https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy 

Also, there is a strategy limit on a snapshot space for scaling reasons, more users may show this strategy as a reason to group their strategies into one strategy

Reply:
That is fine, we can roll all this into a nest configuration but I feel we may run up against hard limits just based on the number of chains we need to connect too as we continue to expand.

Reply:
@bonustrack Any thoughts on strategy limits?

Reply:
We shouldn't have strategy that allow to run a unlimited number of strategies, "multichain" strategy was an exception but now that we support multichain natively I recommend to use normal strategy with custom network. For the strategy that require pagination we can set custom handler in our side, you just need to let us know which one is it.

Reply:
Recommendation is to move away from these strategies and to a basic `erc-20-balanceof` using the native multichain support? Ideal goal is to roll up all the different chains vote counts into a single number (I would geuss this is the default case for most projects) and support said final count delegation. 

If that is possible Is there a good resource for migrating from the multichain strategy to natively using snapshots multichain? My google-fu is failing me 😅.

Reply:
@royalaid Yes that's right, you can try adding a voting strategy you will see now there is a select input for network:
<img width="566" alt="image" src="https://user-images.githubusercontent.com/16245250/175355933-94b5e901-bd7e-4a7f-a4af-da9f7dff9185.png">

Reply:
Right but does that allow us to roll it up into a single count like the multichain strategy does currently? It seems like it just support allowing votes from multiple chains but not collecting them under one sum. I am sure this just me not understanding the feature though.

Reply:
@royalaid Result will be sum of all strategies (even if they are on different networks)